x-pack/metricbeat/module/prometheus/remote_write/data.go

```diff
@@ -44,9 +44,16 @@ func remoteWriteEventsGeneratorFactory(base mb.BaseMetricSet) (remote_write.Remo
 	}
 
 	if config.UseTypes {
+		// By default prometheus in remote_write pushes data with the interval of 60s. In order to calculate counter rate we are setting Period to 60secs accordingly
+		duration, err := time.ParseDuration(config.Period.String())
+		if duration < 60*time.Second || err != nil {
```

Comment:
wondering if we need this check now - we already set the default value to 60, if it was intentionally set to the lover value maybe we should keep it

Reply:
I want to avoid user mistake eg. if users go and specifically set period:10s, without the check we will have the same problem.

I would say to keep it

Reply:
agree on it, lets keep it to 5 for now